target/linux/ramips/dts/mt7621_tplink_archer-c6-v3.dts

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This DTS looks mostly identical to the A6-v3. The Ax/Cx series usually use the same board and only differ in the installed firmware.

Move identical nodes to a dtsi (mt7621_tplink_archer-x6-v3.dts) and include it in both devices.

Reply:
I was able to force A6 snapshot build once the C5 mods were working and found no difference. It seems only Signatures are different in the Flash. I was easily able to revert back to C6 by forcing the upgrade!

Reply:
Will make the change.
Thanks!

Reply:
Greatly interested in testing. Have no UART or other unbricking tool so I wonder if this patch is safe to flash.

Reply:
> Greatly interested in testing. Have no UART or other unbricking tool so I wonder if this patch is safe to flash.

Archer C6 has a recovery mechanism, you need to power cycle it a few time in middle of booting to simulate boot failure and then you can upload the stock firmware using the interface on 192.168.0.1.

But, please, be aware that the router can brick even during the stock firmware upgrade!